.github/workflows/regression_ci.yml

```diff
@@ -80,8 +79,9 @@ jobs:
 
       # Upload the performance output artifacts. This runs even if run diff test fails so debug files can be accessed
       - name: Upload artifacts
-        if: ${{ always() }}  
+        if: ${{ always() }}
         uses: actions/upload-artifact@v4
         with:
           name: regression_artifacts
           path: tests/regression/target/regression_artifacts
+          
```

Comment:
Nit: Not sure what github is complaining about. Maybe you accidentally removed the trailing newline?